### ISSUE TITLE
Update to new ISISDATA variables

### ISSUE THREAD

**Issue by michaelaye** [rbeyer/kalasiris#5]
**Describe the bug**

ISIS has been updated to have a version dependent name for the env variable that points to the ISIS data location:

https://github.com/USGS-Astrogeology/ISIS3#versions-of-the-isis-data-area

**To Reproduce**
* install current ISIS (4.2)
* `from kalasiris import cam2map`

```bash
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-9-a911b11ef4d5> in <module>
----> 1 from kalasiris import cam2map

~/miniconda3/envs/py37/lib/python3.7/site-packages/kalasiris/__init__.py in <module>
      7 __version__ = "1.8.0"
      8 
----> 9 from .kalasiris import *  # noqa: F401,F403
     10 from .k_funcs import *  # noqa: F401,F403
     11 from .Histogram import Histogram  # noqa: F401

~/miniconda3/envs/py37/lib/python3.7/site-packages/kalasiris/kalasiris.py in <module>
     33 # before any calls to ISIS programs.
     34 _isisroot = os.environ["ISISROOT"]
---> 35 _isis3data = os.environ["ISIS3DATA"]
     36 environ = {
     37     "ISISROOT": _isisroot,

~/miniconda3/envs/py37/lib/python3.7/os.py in __getitem__(self, key)
    679         except KeyError:
    680             # raise KeyError with the original key value
--> 681             raise KeyError(key) from None
    682         return self.decodevalue(value)
    683 

KeyError: 'ISIS3DATA'
```

**Expected behavior**

Not sure, but I think it should now also check for `ISISDATA` ?

**Comment by rbeyer** [comment]
Yes, it sure should.  I'll work on a fix.

**Comment by rbeyer** [comment]
It is now fixed in 1.9.0, uploaded to PyPI, conda-forge will update soon.
